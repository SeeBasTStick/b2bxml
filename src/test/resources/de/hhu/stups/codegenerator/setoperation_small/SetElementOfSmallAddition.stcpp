int main() {
    SetElementOfSmall set;
    set.simulate();
    cout << set.getCounter();
    return 0;
}