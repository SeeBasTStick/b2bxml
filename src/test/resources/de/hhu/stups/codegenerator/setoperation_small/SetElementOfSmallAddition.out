public static void main(String[] args) {
    SetElementOfSmall set = new SetElementOfSmall();
    set.simulate();
    System.out.println(set.counter);
}